apps/mobile/app/screens/Authenticated/SettingScreen/components/SingleInfo.tsx

```diff
@@ -47,29 +48,30 @@ const SingleInfo: FC<Props> = ({ title, value, onPress, onDetectTimezone }) => {
 					<AntDesign name="right" size={24} color="#938FA4" />
 				</TouchableOpacity>
 			) : (
-				<TouchableOpacity
-					style={[
-						styles.toggle,
-						dark ? { backgroundColor: "#1D222A" } : { backgroundColor: "#E7E7EA" },
-					]}
-					onPress={() => (onPress ? onPress() : {})}
-				>
-					{dark ? (
-						<View style={styles.iconsContainer}>
-							<SvgXml xml={sunDarkLarge} />
-							<SvgXml xml={moonDarkLarge} />
-						</View>
-					) : (
-						<View style={styles.iconsContainer}>
-							<View style={[styles.iconWrapper, { backgroundColor: "white" }]}>
-								<SvgXml xml={sunLightLarge} />
-							</View>
-							<View style={[styles.iconWrapper, { backgroundColor: "transparent" }]}>
-								<SvgXml xml={moonLightLarge} />
-							</View>
-						</View>
-					)}
-				</TouchableOpacity>
+				<ToggleThemeAnimation toggleTheme={onPress} isDark={dark} />
+				// <TouchableOpacity
+				// 	style={[
```

Comment:
@desperado1802 can we remove all the commented code in this PR if it won't be used in future?